main.ts

```diff
@@ -0,0 +1,150 @@
+let gameBoard: number[][] = [];
+let gameSize: number = parseInt(prompt("Specify size of Board"));
+let winCon: number = parseInt(prompt("Specify how many in a row to win."));
+while (gameSize<winCon || (gameSize === NaN || winCon === NaN)) {
```

Comment:
You might want to take into account case where user do not enter any value ("blank" input) and zero ("0") value